README.md

```diff
@@ -1,2 +1,76 @@
 # brandingUpdate
 
+The table below provides a summary of the terms that are changing in accordance with the recent branding guidelines. When mentioned in the documentation, all of the terms listed below should be updated to their rebranded counterparts as generic capabilities of Adobe Experience Platform. These terms are not proper nouns, and therefore should not be capitalized. Instead, they should be written in lower case like other generic Platform capabilities like "schemas", "profiles", and so on. Rather than being capitalized, branded services, the new terms are styled as generic capabilities of Adobe Experience Platform.
```

Comment:
Drop the last sentence--it's not providing any new information and is just repeating what the paragraph explained.